flume-ng-sinks/flume-ng-kafka-sink/src/test/java/org/apache/flume/sink/kafka/util/TestUtil.java

```diff
@@ -57,6 +57,7 @@ private void init() {
     // get the localhost.
     try {
       hostname = InetAddress.getLocalHost().getHostName();
+      hostname = "localhost";
```

Comment:
Please remove this. This is probably due to an incorrect /etc/hosts file in your test environment